docs/usage/usage.md

```diff
@@ -157,7 +157,10 @@ The worker configuration contains:
 
   **Note**: 
   * A rolling upgrade of the worker group would be triggered in case the `acceleratorType` or `count` is updated.
-  * Some machineTypes like [a2 family](https://cloud.google.com/blog/products/compute/announcing-google-cloud-a2-vm-family-based-on-nvidia-a100-gpu) come with already attached gpu of `a100` type and pre-defined count. If your workerPool consists of those machineTypes, please **do not** specify any GPU configuration.
+  * Some machineTypes like [a2 family](https://cloud.google.com/blog/products/compute/announcing-google-cloud-a2-vm-family-based-on-nvidia-a100-gpu) come with already attached gpu of `a100` type and pre-defined count. If your workerPool consists of such machineTypes, please specify exact GPU configuration for the machine type as specified in Google cloud documentation.  `acceleratorType` to use for families with attached gpu are stated below:
+    1) *a2 family* -> `nvidia-tesla-a100`
+    2) *g2 family* -> `nvidia-L4`
```

Comment:
Are you sure about the `L4`? 
If I remember these are case-sensitive and the error message was mentioning `l4`. Can you doublecheck this @himanshu-kun  ?

Reply:
its lower-case 😬 .. thanks for catching it.

Reply:
updated. pls take a look